pytube/extract.py

```diff
@@ -8,7 +8,8 @@
 from urllib.parse import parse_qs, quote, urlencode, urlparse
 
 from pytube.cipher import Cipher
-from pytube.exceptions import HTMLParseError, LiveStreamError, RegexMatchError
+from pytube.exceptions import HTMLParseError, LiveStreamError, \
```

Comment:
When importing multiple exceptions from a module like pytube.exceptions, it's often considered more readable and maintainable to import them on separate lines rather than using backslashes for line continuation. Here's a cleaner way to import these exceptions
`from pytube.exceptions import (
    HTMLParseError,
    LiveStreamError,
    RegexMatchError,
    ExtractError
)
`

Reply:
Hi, sorry for a delay! Fixed import style in the second commit.

Reply:
This is done @Ripax PTAL.